README.md

```diff
@@ -1,141 +1,164 @@
+</div>
 
-  - [What is Covalent?](https://www.covalent.xyz/what-is-covalent/)
-  - [Covalent in the era of cloud-HPC](https://www.covalent.xyz/navigating-the-modern-hpc-landscape/)
-  - [Basic Concepts of Covalent](https://docs.covalent.xyz/docs/user-documentation/concepts/covalent-basics)
-  - [How does Covalent work?](#how-does-it-work)
-</td>
-    <td>
+---
 
-  - [Covalent with pytorch for classical machine learning](https://docs.covalent.xyz/docs/user-documentation/tutorials/mnist)
-  - [Covalent with pennylane for quantum machine learning](https://docs.covalent.xyz/docs/user-documentation/tutorials/quantumembedding)
-  - [Covalent with Qiskit for quantum-SVM](https://docs.covalent.xyz/docs/user-documentation/tutorials/svm/)
-  - [Covalent with Dask for scaling Ensemble classification](https://docs.covalent.xyz/docs/user-documentation/tutorials/ensemble/)
-  - [Covalent for Deep Neural Network on AWS](https://docs.covalent.xyz/docs/user-documentation/tutorials/dnn_comparison/)
-</td>
- </tr>
-</table>
+### Examples
 
+<div mt="1000">
+<a href="https://docs.covalent.xyz/docs/user-documentation/tutorials/generativeai/"><img src="./doc/source/_static/ai_tutorial.svg" alt="divider"></a>
+<a href="https://docs.covalent.xyz/docs/user-documentation/tutorials/mnist/"><img src="./doc/source/_static/mnist_tutorial.svg" alt="divider"></a>
+<a href="https://docs.covalent.xyz/docs/user-documentation/tutorials/quantumchemistry/"><img src="./doc/source/_static/quantum_tutorial.svg" alt="divider"></a>
+ </div>
 
-## How Does It Work?
-<img src="https://raw.githubusercontent.com/AgnostiqHQ/covalent/master/doc/source/_static/cova_archi.png" align="right" width="40%" alt="Covalent Architecture"/>
-Covalent has three main components:
+---
 
-- A Python module containing an API that you use to build manageable workflows out of new or existing Python functions.
-- A set of services that run locally or on a server to dispatch and execute workflow tasks.
-- A browser-based UI from which to manage workflows and view results.
+<div >
 
-You compose workflows using the Covalent API by simply adding a single line of python decorator and submit them to the Covalent server. The server analyzes the workflow to determine dependencies between tasks, then dispatches each task to its specified execution backend. Independent tasks are executed concurrently if resources are available.
 
-The Covalent UI displays the progress of each workflow at the level of individual tasks.
+### Installation
 
-<details>
-<summary> <h3> The Covalent API </h3> </summary>
-The Covalent API is a Python module containing a small collection of classes that implement server-based workflow management. The key elements are two decorators that wrap functions to create managed *tasks* and *workflows*.
+Covalent is developed using Python version 3.8 on Linux and macOS. The easiest way to install Covalent is using the PyPI package manager.
```

Comment:
> Covalent is developed using Python version 3.8 on Linux and macOS. 

I'd personally suggest removing the version number. The supported versions are in the badge at the top of the README, and of course Covalent supports versions other than 3.8 (i.e. 3.9 and 3.10 right now).

Reply:
I agree, @sriranjanivenkatesan 🙏